static/js/ktl-component/why-teach/index.jsx

```diff
@@ -0,0 +1,663 @@
+                    25 of the top 100 universities in the Times Higher Education World University Rankings 2021 include
+                    Kotlin in their courses. We know of 190 universities that teach Kotlin, which is almost twice as
+                    many as
+                    there were in 2020.
+                  </p>
+
+                  <p className="ktl-text-1 ktl-offset-bottom-m">
+                    Kotlin is used to teach a variety of topics, including object-oriented and functional programming,
+                    software engineering, introductory programming, mobile application development, concurrent
+                    programming,
+                    and scientific programming (Source: internal Teaching Kotlin Study).
+                  </p>
+
+                  <QuotesSlider quotes={[
+                    {
+                      title: 'Eamonn De Leastar, Waterford Institute of Technology',
+                      text: 'On numerous courses, where we proceed through Java to Kotlin, we are considering a Kotlin-first approach.'
+                    },
+                    {
+                      title: 'Eugeniy Tyumentcev, Omsk State University',
+                      text: 'I teach Software Engineering with Kotlin. We also have a separate Android Development course. So I can teach in a language that students can use in other courses.'
+                    },
+                  ]}
+                  />
+                </div>
+
+                <div className="quote-section__info">
+                  <p className="ktl-hero ktl-offset-bottom-xs">25</p>
+                  <p className="ktl-text-2 ktl-offset-bottom-s ktl-dimmed-text">
+                    of the top 100 universities in the Times Higher Education World University Rankings 2021 include
+                    Kotlin
+                    in
+                    their courses.
+                  </p>
+
+                  <a
+                    href="https://docs.google.com/spreadsheets/d/1p77WHo--mxewmxINWMLaTPGXvnEr0JGxgSMcX6C0b_0/edit?usp=sharing"
+                    target="_blank" className="ktl-text-2 ktl-link">
+                    <span className="rs-link">List of universities</span>
+                    <span>&nbsp;↗</span>
+                  </a>
+                </div>
+              </div>
+            </section>
+
+
+            <section className="quote-section ktl-offset-bottom-xl ktl-with-anchor" id="language-of-the-industry">
+              <h3 className="ktl-h3 ktl-offset-bottom-s">
+                Language of the industry
+              </h3>
+
+              <div className="quote-section__grid">
+                <div className="quote-section__content">
+
+                  <p className="ktl-text-1 ktl-offset-bottom-m">
+                    <ul>
+                      <li className="ktl-offset-bottom-s">
+                        — Kotlin is used by top companies such as Google, Twitter, Reddit, Netflix, Uber, BMW, Coursera,
```

Comment:
Actually, the good practise is use bullets outside. Can we use [Typography/Lists](https://rescui.dev/#/Typography/Lists)? Designer agree with that.

Reply:
Agree, will fix